examples/different-payload-types/different_payload_types.bal

```diff
@@ -3,63 +3,63 @@ import ballerina/io;
 import ballerina/mime;
 
-//Client endpoint.
+//[Client](https://ballerina.io/learn/api-docs/ballerina/http/clients/Client.html) endpoint.
 http:Client clientEP = new ("http://localhost:9091/backEndService");
 
 //Service to test HTTP client remote functions with different payload types.
 service actionService on new http:Listener(9090) {
 
     resource function messageUsage(http:Caller caller, http:Request req) {
 
-        //GET remote function without any payload.
+        //[GET](https://ballerina.io/learn/api-docs/ballerina/http/clients/Client.html#get) remote function without any payload.
         var response = clientEP->get("/greeting");
         handleResponse(response);
 
-        //GET remote function with request as message.
+        //[GET](https://ballerina.io/learn/api-docs/ballerina/http/clients/Client.html#get) remote function with request as message.
         http:Request request = new;
         response = clientEP->get("/greeting", message = request);
         handleResponse(response);
 
-        //POST remote function without any payload.
+        //[POST](https://ballerina.io/learn/api-docs/ballerina/http/clients/Client.html#post) remote function without any payload.
         response = clientEP->post("/echo", ());
         handleResponse(response);
 
-        //POST remote function with text as payload.
+        //[POST](https://ballerina.io/learn/api-docs/ballerina/http/clients/Client.html#post) remote function with text as payload.
         response = clientEP->post("/echo", "Sample Text");
         handleResponse(response);
 
-        //POST remote function with `xml` as payload.
+        //[POST](https://ballerina.io/learn/api-docs/ballerina/http/clients/Client.html#post) remote function with `xml` as payload.
         response = clientEP->post("/echo", xml `<yy>Sample Xml</yy>`);
         handleResponse(response);
 
         //POST remote function with `json` as payload.
         response = clientEP->post("/echo", {name: "apple", color: "red"});
         handleResponse(response);
 
-        //POST remote function with `byte[]` as payload.
+        //[POST](https://ballerina.io/learn/api-docs/ballerina/http/clients/Client.html#post) remote function with `byte[]` as payload.
         string textVal = "Sample Text";
         byte[] binaryValue = textVal.toBytes();
         response = clientEP->post("/echo", binaryValue);
         handleResponse(response);
 
-        //Get a byte channel to a given file.
+        //[Get a byte channel](https://ballerina.io/learn/api-docs/ballerina/io/functions.html#openReadableFile) to a given file.
         var bChannel = io:openReadableFile("./files/logo.png");
 
         if (bChannel is io:ReadableByteChannel) {
-            //POST remote function with byte channel as payload. Since the file path is static
+            //[POST](https://ballerina.io/learn/api-docs/ballerina/http/clients/Client.html#post) remote function with byte channel as payload. Since the file path is static
```

Comment:
```suggestion
            //[POST](https://ballerina.io/learn/api-docs/ballerina/http/clients/Client.html#post) remote function with a byte channel as the payload. Since the file path is static,
```